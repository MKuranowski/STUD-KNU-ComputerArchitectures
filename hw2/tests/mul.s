addi x20, x0, 1961
addi x21, x0, 83
addi x22, x0, -13
addi x23, x0, -27
div x1, x20, x21
rem x2, x20, x21
mul x3, x20, x21
div x4, x20, x22
rem x5, x20, x22
mul x6, x20, x22
div x7, x23, x22
rem x8, x23, x22
mul x9, x23, x22
div x10, x22, x21
rem x11, x22, x21
mul x12, x22, x21
