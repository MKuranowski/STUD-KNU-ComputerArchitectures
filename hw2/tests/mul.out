Processor's clock cycles: 16
>>>>>>>>[REGISTER DUMP]<<<<<<<
PC: = 64
x00 = 0
x01 = 23
x02 = 52
x03 = 162763
x04 = -150
x05 = 11
x06 = -25493
x07 = 2
x08 = -1
x09 = 351
x10 = 0
x11 = -13
x12 = -1079
x13 = 0
x14 = 0
x15 = 0
x16 = 0
x17 = 0
x18 = 0
x19 = 0
x20 = 1961
x21 = 83
x22 = -13
x23 = -27
x24 = 0
x25 = 0
x26 = 0
x27 = 0
x28 = 0
x29 = 0
x30 = 0
x31 = 0
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
>>>>>>>>[MEMORY DUMP]<<<<<<<<<
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
